commands/test.js

```diff
@@ -14,48 +14,54 @@ var matchOpts = { matchBase: true };
 function loadTestClasses(client, opts) {
 
   return new Promise(function(resolve, reject) {
-    var nsArg = (opts.ns) ? 
-      '\'' + opts.ns + '\'' : 'null';
+    var nsArg = (opts.ns) ? '\'' + opts.ns + '\'' : null;
 
     var query = [
-      'SELECT Id, Name, SymbolTable',
-      'FROM ApexClass',
-      (opts.all) ? '' : 'WHERE NamespacePrefix = ' + nsArg,
-      'ORDER BY Name'
+      'SELECT Id, Name, SymbolTable, IsValid',
+      '\nFROM ApexClass',
```

Comment:
@kevinohara80 merely made debugging easier.  If they bother you, I have no issue in reverting this.  One thing I did want to change was querying for null namespaces in the event the --ns opt wasn't submitted, but that's not really a big deal one way or the other.